lib/Doctrine/Migrations/Configuration/Configuration.php

```diff
@@ -43,7 +43,7 @@ class Configuration
     private $migrationsColumnName = 'version';
 
     /** @var int */
-    private $migrationsColumnLength = 255;
+    private $migrationsColumnLength = 14;
```

Comment:
What is your reasoning for wanting to do it that way?

Reply:
Magic numbers are code smells I try to avoid by always showing what they represent, be it by using a constant, or showing what computation gave that result. For instance, I would use  `60 * 60 * 24` *  instead of `86400`

Reply:
Ok, I made the change.